src/kbmod/search/common.h

```diff
@@ -43,45 +41,34 @@ namespace search {
 
     // I can't believe string::format is not a thing until C++ 20
     const std::string to_string() const {
-      return "lh: " + std::to_string(lh) +
-        " flux: " + std::to_string(flux) +
-        " x: " + std::to_string(x) +
-        " y: " + std::to_string(y) +
-        " vx: " + std::to_string(vx) +
-        " vy: " + std::to_string(vy) +
-        " obs_count: " + std::to_string(obs_count);
+        return "lh: " + std::to_string(lh) + " flux: " + std::to_string(flux) + " x: " + std::to_string(x) +
+               " y: " + std::to_string(y) + " vx: " + std::to_string(vx) + " vy: " + std::to_string(vy) +
+               " obs_count: " + std::to_string(obs_count);
     }
 
     // returns a yaml-compliant string
     const std::string to_yaml() const {
-      return "{lh: " + std::to_string(lh) +
-        ", flux: " + std::to_string(flux) +
-        ", x: " + std::to_string(x) +
-        ", y: " + std::to_string(y) +
-        ", vx: " + std::to_string(vx) +
-        ", vy: " + std::to_string(vy) +
-        ", obs_count: " + std::to_string(obs_count)
-        +"}";
+        return "{lh: " + std::to_string(lh) + ", flux: " + std::to_string(flux) +
+               ", x: " + std::to_string(x) + ", y: " + std::to_string(y) + ", vx: " + std::to_string(vx) +
+               ", vy: " + std::to_string(vy) + ", obs_count: " + std::to_string(obs_count) + "}";
     }
-  };
+};
 
-  // The position (in pixels) of a trajectory.
-  struct PixelPos {
+// The position (in pixels) of a trajectory.
+struct PixelPos {
     float x;
     float y;
 
-    const std::string to_string() const {
-      return "x: " + std::to_string(x) + " y: " + std::to_string(y);
-    }
+    const std::string to_string() const { return "x: " + std::to_string(x) + " y: " + std::to_string(y); }
 
     const std::string to_yaml() const {
-      return "{x: " + std::to_string(x) + " y: " + std::to_string(y) + "}";
+        return "{x: " + std::to_string(x) + " y: " + std::to_string(y) + "}";
```

Comment:
the `to_yaml` above uses a comma between fields. Would be nice to be consistent.

Reply:
I'm going to keep as-is for now to keep this a purely mechanical application of clang-format, but will change this in the next PR.